tests/integration/relations/opensearch_provider/test_opensearch_provider.py

```diff
@@ -187,6 +187,7 @@ async def test_scaling(ops_test: OpsTest):
     """Test that scaling correctly updates endpoints in databag.
 
     scale_application also contains a wait_for_idle check, including checking for active status.
+    Idle_period checks must be greater than 1 minute to guarantee update_status fires correctly.
```

Comment:
What happens if the update_status hook runs every 6h?

Reply:
This comment refers specifically to how often the test should check status; since we only fire update_status once every minute, we need to check for slightly longer than that to guarantee that the status check fires, and therefore we're definitely checking the up-to-date status information from the charm.

The departing hooks should now handle the status update correctly, but the update_endpoints function is still called on update_status as a catchall. I could test removing this function call to simulate a really long update_status if you wanted, but I feel that it would be a mistake to ever configure update_status to run this infrequently.